.github/workflows/soroban-rpc.yml

```diff
@@ -29,8 +29,11 @@ jobs:
       - uses: ./.github/actions/setup-go
         with:
           go-version: ${{ matrix.go }}
-      - run: |
+      - name: Build soroban contract fixtures
+        run: |
           rustup update
+          rustup target add wasm32-unknown-unknown
+          make build-test-wasms
```

Comment:
I'd place this one is a separate step, since it's far more likely that this one would break compared to the rustup ones..

Reply:
We do it like that in the integration tests and we haven't had a problem